modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -213,37 +213,48 @@ case, *Source IP* refers to the client making the request.
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Value |Description
-| *Max connections*
+| *Max Connections*
 | The maximum amount of simultaneous connections to allow.
 
 *Default value*: 512 connections
 
-| *Max requests per connection*
+| *Max Requests per Connection*
 | The maximum number of requests per connections to allow. +
 This value will determine how much reuse a connection can allow; consider the amount of CPU required to terminate and re-establish a TLS encrypted connection when lowering this value.
```

Comment:
present tense:
```suggestion
Because this value determines how much reuse a connection allows, consider the amount of CPU required to terminate and reestablish a TLS-encrypted connection when lowering this value.
```